jdat_notebooks/soss-transit-spectroscopy/HAT-P-1b-PART3-data-analysis.ipynb

```diff
@@ -4,10 +4,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Analyzing the JWST Pipeline products of HAT-P-1b observations with NIRISS/SOSS\n",
-    "--------------------------------------------------------------\n",
-    "**Author**: Néstor Espinoza (nespinoza@stsci.edu) | **Latest update**: August 23, 2020.\n",
-    "\n",
```

Comment:
The author's name and latest update information were removed here and aren't at the end. This is another thing that would be nice to standardize throughout the project – some notebooks have them at the top, some at the bottom, others not at all.

Reply:
I tried to remove the name everywhere possible. If I missed some, I'll have to go back. This is a philosophical issue about whether we want author names out there or not. I'll need to speak with the other PO's.

Reply:
OK, we can leave it for another time.